configs/components/recom/recom.yaml

```diff
@@ -45,123 +45,119 @@ scenario: "preindustrial"
 
 time_step: 1800
 
-bgc_num: 22
-
-
```

Comment:
They define it in their awiesm.yaml so it should be save to delete?:
https://github.com/esm-tools/esm_tools/blob/0e65d3b2587eb18628845a64a9053dee622571cc/configs/setups/awiesm/awiesm.yaml#L340-L344

Explanation: Unfortunately fesom1 and recom are not coded so that an arbitrary number can be used as `bgc_num`. By default, the values coming from the `namelist.config`s from the recom repo should be used.